test/validate/ferc1_test.py

```diff
@@ -59,76 +58,109 @@ def test_record_id_dupes(pudl_engine, table_name):
 
 
 @pytest.mark.parametrize(
-    "df_name,cols",
+    "asset_key,cols",
     [
-        ("fbp_ferc1", "all"),
-        ("fuel_ferc1", "all"),
-        ("plant_in_service_ferc1", "all"),
-        ("plants_all_ferc1", "all"),
-        ("plants_hydro_ferc1", "all"),
-        ("plants_pumped_storage_ferc1", "all"),
-        ("plants_small_ferc1", "all"),
-        ("plants_steam_ferc1", "all"),
-        ("pu_ferc1", "all"),
-        ("purchased_power_ferc1", "all"),
+        ("out_ferc1__yearly_steam_plants_fuel_by_plant_sched402", "all"),
+        ("out_ferc1__yearly_steam_plants_fuel_sched402", "all"),
+        ("out_ferc1__yearly_plant_in_service_sched204", "all"),
+        ("out_ferc1__yearly_all_plants", "all"),
+        ("out_ferc1__yearly_hydroelectric_plants_sched406", "all"),
+        ("out_ferc1__yearly_pumped_storage_plants_sched408", "all"),
+        ("out_ferc1__yearly_small_plants_sched410", "all"),
+        ("out_ferc1__yearly_steam_plants_sched402", "all"),
+        ("_out_ferc1__yearly_plants_utilities", "all"),
+        ("out_ferc1__yearly_purchased_power_and_exchanges_sched326", "all"),
     ],
 )
-def test_no_null_cols_ferc1(pudl_out_ferc1, live_dbs, cols, df_name):
+def test_no_null_cols_ferc1(live_dbs, asset_value_loader, cols, asset_key):
     """Verify that output DataFrames have no entirely NULL columns."""
     if not live_dbs:
         pytest.skip("Data validation only works with a live PUDL DB.")
     pv.no_null_cols(
-        pudl_out_ferc1.__getattribute__(df_name)(), cols=cols, df_name=df_name
+        asset_value_loader.load_asset_value(asset_key), cols=cols, df_name=asset_key
     )
 
 
+# TODO 2024-09-20: many of these row counts haven't actually been confirmed to
+# be "correct", they're just "what the values were when we added them to the
+# test." Those tables are marked as "unverified" below
```

Comment:
in a very real way i think the main way we have validated these in the past is just that they stay constant or change in direction and scale that we expect (i.e. if we had 10 records for 10 years of data and we add a new year we should get 11 records). So while i like the idea of doing some more nuanced validation here, i'm not sure this unverified-ness makes a whole lot of a difference.